pandas/tests/indexing/multiindex/test_setitem.py

```diff
@@ -215,25 +193,21 @@ def test_multiindex_assignment(self):
             index=[[4, 4, 8], [8, 10, 12]],
         )
 
-        with catch_warnings(record=True):
-            df.ix[4, "c"] = arr
-            exp = Series(arr, index=[8, 10], name="c", dtype="float64")
-            tm.assert_series_equal(df.ix[4, "c"], exp)
+        df.loc[4, "c"] = arr
+        exp = Series(arr, index=[8, 10], name="c", dtype="float64")
```

Comment:
Would generally prefer long form of expected

Reply:
same. since the .ix tests have probably been around for a while, the style used is probably inconsistent with how we would write today.